admin/package-files/systemd/flocker-agent.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Flocker Node Agent
+
+[Service]
+ExecStart=/usr/bin/flocker-zfs-agent $FLOCKER_NODE_NAME $FLOCKER_CONTROL_NODE
```

Comment:
@itamarst made a suggestion to change this at https://github.com/ClusterHQ/flocker/pull/1179#discussion_r25618137. Is there a reason not to?